src/test/java/com/automation/pages/activityStream/EventPage.java

```diff
@@ -7,16 +7,47 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class EventPage extends AbstractPageBase {
+    // Tester Canan
 
     // Upload elements
     @FindBy (id = "bx-b-uploadfile-blogPostForm_calendar")
-    protected WebElement UploadFilesBtn;
+    protected WebElement uploadFilesIcon;
+
+    @FindBy (name = "bxu_files[]")
+    protected WebElement uploadFilesAndImages;
+
+    @FindBy (className = "wd-fa-add-file-light-title-text")
+    protected WebElement downloadFromExternalDrive;
+
+    @FindBy (xpath = "(//*[text()='Select document from Bitrix24'])[3]")
+    protected WebElement selectDocumentFromBitrix;
+
+    @FindBy (xpath = "//*[text()='Recent items']")
+    protected WebElement recentItems;
+
+    @FindBy (xpath = "//div[@class='bx-file-dialog-content-wrap']//div//span//a")
+    protected WebElement firstItem;
+
+    @FindBy (xpath = "//span[text()='My Drive']")
+    protected WebElement myDrive;
+
+    @FindBy (xpath = "//a[text()='Uploaded files']")
+    protected WebElement uploadedFiles;
```

Comment:
This WebElement name also already exists, again is it linking to the same location?
`@FindBy(name = "bxu_files[]")	
    private WebElement uploadfilesandimages;`

Reply:
Yes, it is pointing the same. I did not recognize that it was already there.